graphix/pattern.py

```diff
@@ -801,11 +802,7 @@ def _measurement_order_depth(self):
             optimal measurement order for parallel computing
         """
         d, l_k = self.get_layers()
-        meas_order = []
-        for i in range(d):
-            for node in l_k[i]:
-                meas_order.append(node)
-        return meas_order
+        return list(itertools.chain.from_iterable(l_k[i] for i in range(d)))
```

Comment:
The following code is closer to the code in the subsequent function `get_measurement_order_from_gflow`, and appears to be slightly faster.
```suggestion
        meas_order = []
        for i in range(d):
            meas_order.extend(l_k[i])
        return meas_order
```